src/VisualStudio/Core/Def/Implementation/MoveStaticMembers/MoveStaticMembersDialog.xaml.cs

```diff
@@ -0,0 +1,63 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#nullable disable
+
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Input;
+using Microsoft.VisualStudio.PlatformUI;
+
+namespace Microsoft.VisualStudio.LanguageServices.Implementation.MoveStaticMembers
+{
+    /// <summary>
+    /// Interaction logic for MoveMembersToTypeDialog.xaml
+    /// </summary>
+    internal partial class MoveStaticMembersDialog : DialogWindow
+    {
+        public string MoveStaticMembersDialogTitle => "[WIP] Move Static Members";
```

Comment:
We can go ahead an add an entry (`Move_Static_Members`) to the ServicesVSResources.resx resource file for this string.